pull-requests/readme.md

```diff
@@ -2,4 +2,5 @@
 
 ## Sections within Pull Requests
 
+* [Pull Request Template](./PullRequestTemplate.md)
```

Comment:
Please rename to lower case eg: pull-request-template.md. And also please add a link to this template from the main pull-request page https://github.com/microsoft/code-with-engineering-playbook/tree/master/pull-requests